Eatery/Controllers/Eateries/CampusEateriesViewController.swift

```diff
@@ -77,6 +77,9 @@ class CampusEateriesViewController: EateriesViewController {
 
     private func showMenu(of eatery: CampusEatery, animated: Bool) {
         let menuViewController = CampusEateryMenuViewController(eatery: eatery, delegate: self, userLocation: userLocation)
+        let payload: Payload!
```

Comment:
No need to have `Payload` as an implicitly unwrapped optional! Just use the ternary directly like this: 

`let payload = eatery.eateryType == .dining ? CampusDiningCellPressPayload() : CollegetownCellPressPayload`

I'm not sure if `eateryType` includes `.dining` but I'm almost certain there is an enum case that matches what you're looking for. Comparing `rawValue` to a string is rather fragile. We don't want to change the `rawValue` and have our analytics break